### DISCUSSION TITLE
Adding lightning-strike-distance-limit to spigot before 1.13

### DISCUSSION THREAD

**Discussion by ProblemsSender** [EssentialsX/Essentials#4257] [Ideas and Feature Suggestions] [answered]
paper after 1.13 have option lightning-strike-distance-limit
I think spigot before 1.13 should have that option too

**Comment by pop4959** [comment, marked as answer]
Not really sure what you're asking here. If it's an issue with Spigot perhaps you should be reporting this to them instead?

**Reply by ProblemsSender** [reply]
Sorry man, i thought you know about the option lightning-strike-distance-limit in paper.yml
Here let me clarify to you, you know spigot before 1,13 we cannot change the sound and the light effect into a limit distance(even paper before 1.13 had not this added in too). Which is a kind of inconvenience or unrealistic to see thunder effect whenever the player uses/lightning. the sky keeps flashing with the noisy sound nonstop. (at least spigot.yml always have the option to limit wither and ender dragon sound)
